js/luno.js

```diff
@@ -126,6 +126,44 @@ module.exports = class luno extends Exchange {
         return this.parseOrderBook (orderbook, timestamp, 'bids', 'asks', 'price', 'volume');
     }
 
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let response = await this.privateGetOrders (this.extend ({
+            'id': id.toString (),
+        }, params));
+        let state = (response.state == 'PENDING') ? 'open' : 'closed';
+        // switch response.state {
+        //     case 'PENDING':
+        //         state = 'open';
+        //         break;
+        //     case 'COMPLETE':
+        //         state = 'closed'; // **OR cancelled**
+        //         break;
+        // }
+        let side = (response.type == 'ASK') ? 'sell' : 'buy';
```

Comment:
The same assoc key access syntactic issue, it's repeating everywhere over and over, so I won't mention it anymore